### ISSUE TITLE
paymentrequest_pb2.py

### ISSUE THREAD

**Issue by memopo** [qtumproject/qtum-electrum#44]
Trying to run from linux source:
Error: could not find paymentrequest_pb2.py. Create it with 'protoc --proto_path=lib/ --python_out=lib/ lib/paymentrequest.proto'
Ran protoc don't fix the issue. The files exist: 
lib/paymentrequest_pb2.py  lib/paymentrequest.proto  lib/paymentrequest.py

**Comment by icodeface** [comment]
hi, did you follow the instructions https://github.com/qtumproject/qtum-electrum#development-version

**Comment by memopo** [comment]
Yes

**Comment by cupid-stunt** [comment]
Got a major problem with the wallet. I installed the wallet using a Trezor on MACos and around Christmas time I uploaded some Qtum to the wallet. This week I am trying to move the Qtum to a Ledger nano. Everything seems fine until I try and send the transaction where I get a "failed to compile output" warning. I've scoured the net trying to find a solution but even though it is a known issue and attempts to solve it in my case fail. I have installed a new wallet in Windows and with the Trezor, I can see the coins but I still get the same warning. I suspect the Private keys held in the Trezor keeps the coins safe and if I can use another Trezor supported wallet then maybe i'm able to access them. Anyone know of another Trezor supported Qtum wallet?
